calculator/main.py

```diff
@@ -13,10 +21,24 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
+    
+    
 
 
 def perform_operation(calc, operation, params):
+    if "hello" in params:
```

Comment:
Not a good idea to have a check only for a specific word ... should have a check for each element of params to be of type int or float